Makefile

```
@@ -0,0 +1,205 @@
+	@echo "$(GREEN)Examples:$(RESET)"
+	@echo "  make all          # Generate docs, format, and lint everything"
+	@echo "  make docs         # Generate documentation only"
+	@echo "  make lint         # Run all linters"
+	@echo "  make format       # Format all files"
+	@echo "  make check        # Quick syntax checks"
+
+# Main targets
+all: install-tools docs format lint ## Generate docs, format, and lint everything
+	@echo "$(GREEN)✅ All tasks completed successfully$(RESET)"
+
+docs: ## Generate documentation for all actions
```

checkmake (0.2.2)

[warning] Target body for "docs" exceeds allowed length of 5 (22).

(maxbodylength)